README.md

```diff
@@ -1,2 +1,25 @@
 # zsh-pyenv-lazy
-A zsh plugin for lazy loading of pyenv. The initial `eval "$(pyenv init -)"` is executed the first time `pyenv` is called.
+
+A zsh plugin for lazy loading of pyenv.
+The initial `eval "$(pyenv init -)"` is executed the first time `pyenv` is called.
+
+If `ZSH_PYENV_LAZY_VIRTUALENV` is set to `true`, also call `eval "$(pyenv virtualenv-init -)"`.
+
+## Installation
+
+### oh-my-zsh
+
+```sh
+git clone https://github.com/davidparsson/zsh-pyenv-lazy.git ~/.oh-my-zsh/custom/plugins/pyenv-lazy
+```
+
+## Usage
+
+In your `~/.zshrc`, add `pyenv-lazy` to your `plugins` variable.
```

Comment:
I think this is a part of the installation for `oh-my-zsh` as well, right?

Reply:
Yes, that may be oh-my-zsh specific. I'm new to zsh!

Reply:
That's fine, I've fixed that on master. Thanks for your contribution!